.github/workflows/find-inactive-tsc.yml

```diff
@@ -0,0 +1,46 @@
+name: Find inactive TSC members
+
+on:
+  schedule:
+    # Run every Tuesday 12:05 AM UTC.
+    - cron: '5 0 * * 2'
+
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 16.x
+
+jobs:
+  find:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+
+      - name: Clone nodejs/TSC repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          repository: nodejs/TSC
+          path: .tmp
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Find inactive TSC members
+        run: tools/find-inactive-tsc.mjs
+
+      - name: Open pull request
+        uses: gr2m/create-or-update-pull-request-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
+        with:
+          author: Node.js GitHub Bot <github-bot@iojs.org>
+          body: This PR was generated by tools/find-inactive-tsc.yml.
+          commit-message: "meta: move one or more TSC members to emeritus"
+          labels: meta
+          title: "meta: move one or more TSC members to emeritus"
```

Comment:
Do we need to specify a branch like `find-inactive-collaborators.yml` does?

Reply:
Yes we do. Done. Thanks!